README.md

```diff
@@ -36,30 +36,32 @@ OPTIONAL SECTIONS:
   - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
+- [🤝 Contributors](#contributors)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [setting a repo module 1] <a name="Portfolio Mobile first version"></a>
+# 📖 Portfolio Mobile First <a name="Portfolio Mobile first version"></a>
```

Comment:
[OPTIONAL] Nice job using the README template, I'd suggest you avoid removing the `a` tags content, because they link each section to their right `Table of Contents` item, if you change the a tag content, make sure to also edit the `Table of Contents`